tests/kuttl/test-suites/default/common/cleanup-errors.yaml

```diff
@@ -131,3 +131,17 @@ kind: Pod
 metadata:
   labels:
     service: watcher-applier
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: watcher-kuttl-decisionengine
+  labels:
+    service: watcher-decisionengine
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    service: watcher-decisionengine
+  name: watcher-kuttl-decisionengine-0
```

Comment:
I would say that only matching the service label should be enough to fail

Reply:
ack sure will make it more generic by removing name